open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/client/nodes/DefaultSubjectAreaNodeClients.java

```diff
@@ -24,19 +20,17 @@
 /**
  * The OMAS client library implementation of the Subject Area OMAS.
  * This interface provides entities {@link Node} authoring interface for subject area experts.
- * A standard set of customers is described in {@link SubjectAreaCategory}, {@link SubjectAreaTerm},
- * {@link SubjectAreaProject}, {@link SubjectAreaGlossary}
  */
-public class SubjectAreaNode implements SubjectAreaCategory, SubjectAreaTerm, SubjectAreaProject, SubjectAreaGlossary {
-    private Map<Class<?>, SubjectAreaNodeClient<?>> cache = new HashMap<>();
-    private static final String DEFAULT_SCAN_PACKAGE = SubjectAreaNode.class.getPackage().getName();
+public class DefaultSubjectAreaNodeClients implements SubjectAreaNodeClients {
+    private final Map<Class<?>, SubjectAreaNodeClient<?>> cache = new HashMap<>();
```

Comment:
It became possible to do it in java 5 because reflection and annotations appeared in the 5 version. =) So it is not later Java capabilities.
But I will try to comment it:)